src/main/java/org/candlepin/util/Util.java

```diff
@@ -331,6 +331,10 @@ public static List<String> getPossibleUuids(String... ids) {
         return results;
     }
 
+    public static List<String> getPossibleUuids(String... ids) {
+        return getPossibleUuids(Arrays.asList(ids));
```

Comment:
nit: there's no null check on the array as a whole. What I usually do for these kind of helper/redirect methods is something like this:

`return getPossibleUuids(ids != null ? Arrays.asList(ids) : null);`

But in this case, we'd also need to add a null check on the other method to handle it correctly. It doesn't today, so fixing that here buys us exactly nothing. If you're up for it, fix it in both places; otherwise, leave it.

Reply:
wouldn't it be better to return an empty list because if we return null it could lead to an error because we can't have this in jpql:

 String jpql = """
            SELECT g.consumer FROM GuestId g
            WHERE g.consumer.owner.id = :ownerId
                AND LOWER(g.guestId) IN :possibleIds
            ORDER BY g.updated DESC
            """;
getEntityManager().createQuery(jpql, Consumer.class)
                .setParameter("ownerId", ownerId)
                .setParameter("possibleIds", null)
                .setMaxResults(1)
                .getSingleResult();

Reply:
done